pkgs/misc/vim-plugins/default.nix

```diff
@@ -3280,7 +3280,7 @@ let
 
   };
 
-} // lib.optionalAttrs (config.allowAliases or true) (with self; {
+} // lib.optionalAttrs (config.allowAliases or false) (with self; {
```

Comment:
We definitely want this to default to true still. Users can set it to false in config.nix.

Reply:
Feel free to revert.

It was mainly to ensure the evaluation is done with ```allowAliases=false``` as  #43692 missed so many replacement yet successfully evaluate.

Although defaulting to false might have sense, putting ```allowAliases``` in a row with ```allowUnfree``` and ```allowBroken``` (they default to false as being "bad", but everyone set them to true in confiig.nix)

Reply:
I cannot do the fix (nor resolve the merge conflict), as the branch became read-only
Perhaps due to the auto-request for review (```"volth requested review from edolstra, Ericson2314, nbp and peti as code owners 10 hours ago"```) it is frozen until they all will review

Reply:
Ok, managed to rebase
I will commit ```s/config.allowAliases or false/config.allowAliases or true/g``` after ofborg eval